### PR TITLE
Review comments from Krzysztof

### PR DESCRIPTION
**Pull request by timj**

### CODE REVIEW

File: ci.tex [hunk 1]
```diff
@@ -27,7 +27,7 @@ \subsubsection{Configuration \& Deployment}
 at the time DM was transitioning away from Buildbot.
 
 \href{https://puppet.com/}{Puppet} was selected as a configuration management (CM) tool as, at that time, it had the most sophisticated Jenkins
-management abilities among the CM tools surveyed via the \href{https://github.com/voxpupuli/puppet-jenkins}{\texttt{puppet-jenkins}} module.
+management abilities among the CM tools surveyed, via the \href{https://github.com/voxpupuli/puppet-jenkins}{\texttt{puppet-jenkins}} module.
```
Comment (kfindeisen) [inline ci.tex]: > Puppet was selected as a configuration management (CM) tool as, at that time, it had the most sophisticated Jenkins management abilities among the CM tools surveyed, via the puppet-jenkins module.

I think this is grammatically incorrect, and it's certainly confusing. Why not reorder to "Puppet, via the puppet-jenkins module, was selected..."?

Comment (ktlim) [reply ci.tex]: Puppet was selected as a configuration management (CM) tool as its puppet-jenkins module provided the most sophisticated Jenkins management abilities among the tools surveyed at that time.

File: codeqa.tex [hunk 2]
```diff
@@ -54,7 +54,7 @@ \subsubsection{Language Standards}
 
 We do not require a specific C++ compiler to build the DM software, but we do require compliance
 with the  ISO C++14 standard\cite{cpp14}. We configure the Clang and GCC compilers to fail if the code is non-compliant.
-There is still code that uses C++11 and older style but we are slowly updating the code as we encounter constructs that can be improved or simplified.
+There is still code that uses C++98 but we are slowly updating the code as we encounter constructs that can be improved or simplified.
```
Comment (pschella) [inline codeqa.tex:56]: +1 on this change.

**Comment by timj** [comment]
@ktlim did you come to any decision with @kfindeisen on the sprint planning comments?

**Comment by kfindeisen** [comment]
I'm fine with the rewording @ktlim proposed on [Jira](https://jira.lsstcorp.org/browse/PUB-64?focusedCommentId=102357#comment-102357).

**Pull request merged**